### DISCUSSION TITLE
GoPro Hero 11 QR Codes Not Reading

### DISCUSSION THREAD

**Discussion by LLTCEnterprises** [gopro/labs#249] [General]
Troubleshooting an issue where my Hero 11 no longer reads QR codes.  Was working with a 3rd party using their GoPro Labs-created QR codes and then the GoPro just stopped reading the QR codes.  Several times going through the factory reset and manual firmware update process yielded no positive results.  Any ideas?

Thanks!

**Comment by dnewman-gpsw** [comment]
Please share the exact firmware version.  The log file from the SD Card within MISC/qrlog.txt

**Reply by LLTCEnterprises** [reply]
"firmware version":"H22.01.02.01.00"
as downloaded from https://gopro.com/en/us/support/hero11-black-product-update/macos

**Reply by dnewman-gpsw** [reply]
That is not Labs firmware.  Download the latest Labs firmware from https://community.gopro.com/s/article/GoPro-Labs

**Reply by LLTCEnterprises** [reply]
"firmware version":"H22.01.02.01.70"
Seems to be reading now.  Thank you!
